src/main/java/org/broadinstitute/hellbender/tools/funcotator/Funcotator.java

```diff
@@ -580,7 +581,8 @@ private void addFeaturesForLocatableDataSource( final Path dataSourceFile,
                         IOUtils.getPath( dataSourceProperties.getProperty(DataSourceUtils.CONFIG_FILE_FIELD_NAME_SRC_FILE) )
                 ).toUri().toString(),
                 name,
-                featureClazz
+                featureClazz,
+                FEATURE_QUERY_LOOKAHEAD
```

Comment:
Can you confirm that Funcotator never "looks back" when querying its `FeatureContext`? If it does, the existing caching scheme might be non-ideal.

Reply:
It looks like it only ever calls it on the same location as the driving variant.  It does some contig translation, but it should always translate the same way so I don't think that's a problem.